[kyverno] Add 1.14 (#7290)
captn3m0/endoflife.date: products/kyverno.md

-   purl: pkg:docker/chainguard/kyverno
-   purl: pkg:oci/kyverno?repository_url=cgr.dev/chainguard

auto:
  methods:
  -   git: https://github.com/kyverno/kyverno.git

# eol(X) = releaseDate(X+3)
# Kubernetes support is documented on https://kyverno.io/docs/installation/#compatibility-matrix.
releases:
-   releaseCycle: "1.14"

>     releaseDate: 2025-04-24
>     eol: false
>     latest: "1.14.0"
>     latestReleaseDate: 2025-04-24

Author: ```suggestion
    releaseDate: 2025-04-25
    eol: false
    latest: "1.14.0"
    latestReleaseDate: 2025-04-25
```

>     supportedK8sVersions: Not yet documented # Not yet on https://kyverno.io/docs/installation/#compatibility-matrix

Author: ```suggestion
    supportedK8sVersions: N/A # Not yet on https://kyverno.io/docs/installation/#compatibility-matrix
```

-   releaseCycle: "1.11"
    releaseDate: 2023-11-10

>     eol: 2025-04-24 # releaseDate(1.14)

Author: ```suggestion
    eol: 2025-04-25 # releaseDate(1.14)
```
